### ISSUE TITLE
Bring back View files in SAF original Innertune download feature

### ISSUE THREAD

**Issue by Chickenshrimps** [Malopieds/InnerTune#436]
### Checklist

- [X] I've checked that there is no other issue about this feature request.
- [X] This issue contains only one feature request.
- [X] The title of this issue accurately describes the feature request.

### Feature description

Feature to bring back original settings for downloading feature Innertune 4.4 to store downloaded songs to internal storage device 

### Why do you want this feature?

To avoid getting loss of data of downloaded songs if app gets accidentally deleted 
Pls remove the download cache feature and add the original feature of storing downloaded songs in SAF internal storage device 

### Additional information

Thank you for your cooperation 
I know I have made several feature requests regarding this topic and you have always show no promise to this feature
Pls reconsider doing this in future updates

**Comment by derei** [comment]
@Malopieds  I agree with @Chickenshrimps .
Currently, one accidental tap, and the entire cache is gone. Tubular handles downloads much better, as actually externally accessible files. We need that in Inner Tunes too, please.

**Comment by Malopieds** [comment]
Please see comment below:

https://github.com/z-huang/InnerTune/issues/1167#issuecomment-2287829367

**Comment by derei** [comment]
@Malopieds makes sense.
Could you please add a solid warning when deleting the cache, pwrhaps? Just being able to accisentally touch it, and having all cache removed, is just too risky.

Thanks!

**Comment by Chickenshrimps** [comment]
> @Malopieds makes sense. Could you please add a solid warning when deleting the cache, pwrhaps? Just being able to accisentally touch it, and having all cache removed, is just too risky.
> 
> Thanks!

**Comment by Chickenshrimps** [comment]
Ok but you can atleast add a auto backup feature or a warning as @derei said, can't you?

**Comment by Malopieds** [comment]
Yes will do, I don't have much time, and I won't have a lot soon...
